homeassistant/components/enphase_envoy/const.py

```diff
@@ -1,6 +1,15 @@
 """The enphase_envoy component."""
+from pyenphase import (
+    EnvoyAuthenticationError,
+    EnvoyAuthenticationRequired,
+)
+
 from homeassistant.const import Platform
 
 DOMAIN = "enphase_envoy"
 
 PLATFORMS = [Platform.SENSOR]
+
+CONF_TOKEN = "token"
+
+INVALID_AUTH_ERRORS = (EnvoyAuthenticationError, EnvoyAuthenticationRequired)
```

Comment:
You also have these defined in config_flow, you can remove them and use these instead

Reply:
You were quicker than me, nice!